adafruit_bus_device/i2c_device.py

```diff
@@ -41,15 +48,15 @@ class I2CDevice:
                 device.write(bytes_read)
     """
 
-    def __init__(self, i2c, device_address, probe=True):
+    def __init__(self, i2c: I2C, device_address: int, probe: bool = True):
 
         self.i2c = i2c
         self.device_address = device_address
 
         if probe:
             self.__probe_for_device()
 
-    def readinto(self, buf, *, start=0, end=None):
+    def readinto(self, buf: bytearray, *, start: int = 0, end: Optional[int] = None):
```

Comment:
Don't we have a `WritableBuffer` type? I think it should work for more than just bytearray.

Reply:
See comment below!